db/models/index.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const fs        = require("fs");
+const path      = require("path");
+const Sequelize = require("sequelize");
+const basename  = path.basename(__filename);
+const env       = process.env.NODE_ENV || "development"; // eslint-disable-line no-process-env
+const config    = require(__dirname + "/../config/config.json")[env];
+const db        = {};
+
+let sequelize;
+
+if (config.use_env_constiable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config); // eslint-disable-line no-process-env
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
```

Comment:
Not sure if this would be easier if we moved this file into the parent directory, ./db/models.js, and then the models/* directory was just the model schemas. Then you could use something like [**glob**](http://npm.im/glob) to do:

```js
glob.sync("./models/*.js")
  .forEach(file => {
    const model = sequelize.import(path.join(__dirname, file));
    db[model.name] = model;
  });
```